tools/cockpit.spec

```diff
@@ -81,6 +81,13 @@ Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{v
 %define build_optional 1
 %endif
 
+# Ship custom SELinux policy only in Fedora and RHEL-9 onward
+%if 0%{?rhel} >= 9 || 0%{?fedora}
+%define selinuxtype targeted
+%define with_selinux 1
+%define selinux_policy_version %(rpm -q --queryformat "%{V}" selinux-policy)
```

Comment:
We cannot build this if there is no selinux-policy installed.
Of course, that is brought by BuildRequires. So maybe having this not as `%define` but directly in condition or something? Or that would still be too early as the rpm is parsed before? So just dummy version?

Reply:
Dummy version seems like a good enough workaround for packit builds indeed. Thanks!